microsoft-edge/progressive-web-apps-chromium/how-to/sidebar.md

```diff
@@ -202,15 +178,13 @@ Follow these recommendations when using a mobile-variant of your app in the side
 
 PWAmp is a music player PWA demo application that can be pinned to the sidebar in Microsoft Edge. To test PWAmp as a sidebar app:
 
-1. Enable sidebar support as described in [Enable sidebar support of PWAs in Microsoft Edge](#enable-sidebar-support-of-pwas-in-microsoft-edge).
-
 1. Open Microsoft Edge and make sure the sidebar is displayed. If the sidebar is not displayed, go to `edge://settings/sidebar` and then turn on the **Always show sidebar** toggle:
 
     ![The Edge Settings page with the "Always show sidebar" setting](./sidebar-images/always-show-sidebar.png)
 
 1. Go to [PWAmp](https://microsoftedge.github.io/Demos/pwamp/).
 
-1. In the sidebar, click **Customize sidebar**, and then click **Add current page**:
+1. In the sidebar, click **Customize**, and then click **Open in sidebar**:
```

Comment:
specify "button" or "link" 2x, see if "select" flows better for 2nd part
Rendered section: 
pending